doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -1149,6 +1146,13 @@ Match on values
       | @tac2pat1 , {*, @tac2pat1 }
       | @tac2pat1
 
+.. tacn:: if @ltac2_expr5 then @ltac2_expr5 else @ltac2_expr5
+   :name: if-then-else (Ltac2)
+
+   Equivalent to a :tacn:`match <match (Ltac2)>` on a boolean value.  If the
+   first :token:`ltac2_expr5` evaluates to true, the second :token:`ltac2_expr5`
+   is evaluated.  Otherwise the third :token:`ltac2_expr5` is evaluated.
```

Comment:
```suggestion
.. tacn:: if @ltac2_expr5__test then @ltac2_expr5__then else @ltac2_expr5__else
   :name: if-then-else (Ltac2)

   Equivalent to a :tacn:`match <match (Ltac2)>` on a boolean value.  If the
   :n:`@ltac2_expr5__test` evaluates to true, :n:`@ltac2_expr5__then`
   is evaluated.  Otherwise :n:`@ltac2_expr5__else` is evaluated.
```

I should have suggested using subscripts, I think it reads better with them.  It will look a bit like this from the Ltac chapter:

![image](https://user-images.githubusercontent.com/1253341/97908158-2ea70980-1cfb-11eb-8bf2-406f72c7c51e.png)